### PR TITLE
Update alumni data

### PR DESCRIPTION
**Pull request by Kn99HN**

### CODE REVIEW

File: src/data/alumni.js [hunk 1]
```diff
@@ -49,4 +49,9 @@ export default {
     description:
       "Alumni from Woodmere, New York studying Computer Science. Outside of BUGS, you can find him coaching hockey at a local high school.",
   },
+  "Ami Nachiappan": {
+    role: "Event Coordinator",
+    description:
+      "Ami is a Senior from California studying Computer Science and minoring in Web Design. In her free time, she loves spending time with family, running, and of course, coding!",
```
Comment (yuchenliu15) [inline src/data/alumni.js]: @Kn99HN maybe change all beginnings to "alumni who ..."?

Comment (esilverm) [reply src/data/alumni.js]: Agreed. The descriptions are in need of an update

**Review by esilverm — COMMENTED** [review]
See Yuchen's comment. Otherwise it is a thumbs up from me

**Comment by Kn99HN** [comment]
What should we do with the `Role`? It's not being used to display on the website but I think it should be there since some were the founders? Or should we integrate that in their description?

**Comment by yuchenliu15** [comment]
@Kn99HN integrating into their descriptions would be really cool

**Review by yuchenliu15 — APPROVED** [review]
LGTM!

**Pull request merged**